frontend/src/pages/Settings/Plex/index.tsx

```diff
@@ -1,23 +1,15 @@
-import { Box, Button, Collapse, Group, Paper, Stack } from "@mantine/core";
-import { useDisclosure } from "@mantine/hooks";
-import { faChevronDown, faChevronUp } from "@fortawesome/free-solid-svg-icons";
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Box, Paper } from "@mantine/core";
 import {
   Check,
   CollapseBox,
   Layout,
-  Message,
-  Number,
   Section,
-  Text,
 } from "@/pages/Settings/components";
 import { plexEnabledKey } from "@/pages/Settings/keys";
-import { PlexSettings } from "./PlexSettings";
+import LibrarySelector from "./LibrarySelector";
+import PlexSettings from "./PlexSettings";
```

Comment:
Maybe we should call the LibrarySelector inside Plexsettings instead to skip the import here?